setup.py

```diff
@@ -60,7 +68,9 @@ def read(*rnames):
       extras_require=dict(
           test=[
               'zope.component [test]',
+              'zope.testrunner',
```

Comment:
This is surprising to me -- we're not importing anything from `zope.testrunner` in our test code.

I suppose we rely on zope.testrunner's test layer support for our tests, but in theory that could be implemented by other test runners as well.